Source/WebCore/editing/ios/EditorIOS.mm

```diff
@@ -307,11 +307,11 @@ static void getImage(Element& imageElement, RefPtr<Image>& image, CachedImage*&
 {
     // FIXME: This is a hacky workaround for the keyboard calling this method too late -
     // after the selection and focus have already changed. See <rdar://problem/5975559>.
-    Element* focused = document().focusedElement();
-    Node* composition = compositionNode();
-    if (composition && focused && focused != composition && !composition->isDescendantOrShadowDescendantOf(focused)) {
```

Comment:
`focused != composition` is true unconditionally since `composition` is a `Text*` and `focused` is an `Element*` and those 2 types are not related.